### PR TITLE
Update command-line-arguments-microsoft-defender-antivirus.md

### LINKED ISSUE #12116: Wrong description to parameters of "ResetPlatform" and "RevertPlatform".
The description of parameters of -ResetPlatform and -RevertPlatform are opposite.

The -ResetPlatform should be used to "reset platform binaries back to %ProgramFiles%\Windows Defender".

and the -RevertPlatform should be used to "Revert platform binaries back to the previous installed version of the Defender platform".

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: fd851389-130d-f800-f73d-b7561ccff612
* Version Independent ID: 58ee1cc9-d047-da22-0f56-865643d10d7c
* Content: [Use the command line to manage Microsoft Defender Antivirus](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/command-line-arguments-microsoft-defender-antivirus?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-endpoint/command-line-arguments-microsoft-defender-antivirus.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/command-line-arguments-microsoft-defender-antivirus.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @denisebmsft
* Microsoft Alias: **deniseb**

### PR DESCRIPTION
**Pull request by msbemba**
Changed description of parameters  "ResetPlatform" and "RevertPlatform"

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12116

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/command-line-arguments-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -70,8 +70,8 @@ In our example, the MpCmdRun utility starts a full antivirus scan on the device.
 |`-RemoveDynamicSignature [-SignatureSetID]`|Removes dynamic security intelligence.|
 |`-CheckExclusion -path <path>`|Checks whether a path is excluded.|
 |`-ValidateMapsConnection`|Verifies that your network can communicate with the Microsoft Defender Antivirus cloud service. This command will only work on Windows 10, version 1703 or higher.|
-|`-ResetPlatform`| Revert platform binaries back to the previous installed version of the Defender platform.|
-|`-RevertPlatform`|reset platform binaries back to `%ProgramFiles%\Windows Defender`.|
+|`-ResetPlatform`| Reset platform binaries back to `%ProgramFiles%\Windows Defender`|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/command-line-arguments-microsoft-defender-antivirus.md]:
```suggestion
|`-ResetPlatform`| Reset platform binaries back to `%ProgramFiles%\Windows Defender`.|
```

File: microsoft-365/security/defender-endpoint/command-line-arguments-microsoft-defender-antivirus.md [hunk 2]
```diff
@@ -70,8 +70,8 @@ In our example, the MpCmdRun utility starts a full antivirus scan on the device.
...
+|`-RevertPlatform`| Revert platform binaries back to the previous installed version of the Defender platform.|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/command-line-arguments-microsoft-defender-antivirus.md]:
```suggestion
|`-RevertPlatform`| Revert platform binaries back to the previously installed version of the Defender platform.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@denisebmsft Please merge this PR. Thanks

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba @JohanFreelancer9

**Pull request merged**